sinks/cache/cache_impl.go

```diff
@@ -21,16 +21,19 @@ import (
 
 	source_api "github.com/GoogleCloudPlatform/heapster/sources/api"
 	"github.com/GoogleCloudPlatform/heapster/store"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
 const rootContainer = "/"
 
 type containerElement struct {
+	lastUpdated time.Time
 	Metadata
 	metrics store.TimeStore
 }
 
 type podElement struct {
+	lastUpdated time.Time
```

Comment:
The unit tests are supposed to cover the corner cases. I intend to add an e2e test for this. But with the time crunch, that might not happen.